### PR TITLE
Aml 3280 aorn page

### PR DESCRIPTION
**Pull request by MichaelYoung1981**

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.Web/Controllers/SearchPensionRegulatorController.cs [hunk 1]
```diff
@@ -89,6 +91,45 @@ public ActionResult SearchPensionRegulator(string hashedAccountId, SearchPension
...
+        [HttpGet]
+        [Route("pensionregulator/aorn")]
+        public async Task<ViewResult> SearchPensionRegulatorByAorn()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        [Route("pensionregulator/aorn")]
+        public async Task<ActionResult> SearchPensionRegulatorByAorn(string aorn, string payeRef)
```
Comment (Howesy77) [inline src/SFA.DAS.EmployerAccounts.Web/Controllers/SearchPensionRegulatorController.cs]: Should we be checking AntiForgeryToken?

Comment (MichaelYoung1981) [reply src/SFA.DAS.EmployerAccounts.Web/Controllers/SearchPensionRegulatorController.cs]: Absolutely!  I've added the attribute.

File: src/SFA.DAS.EmployerAccounts.UnitTests/Queries/GetOrganisationsByAornTests/WhenIValidateTheQuery.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        private GetOrganisationsByAornValidator _validator;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _validator = new GetOrganisationsByAornValidator();
+        }
+
+        [Test]
+        public void ThenTrueIsRetrunedWhenAllFieldsArePopulated()
```
Comment (Howesy77) [inline src/SFA.DAS.EmployerAccounts.UnitTests/Queries/GetOrganisationsByAornTests/WhenIValidateTheQuery.cs]: Spelling mistake

File: src/SFA.DAS.EmployerAccounts/Services/PensionRegulatorService.cs [hunk 3]
```diff
@@ -35,6 +35,15 @@ public async Task<IEnumerable<Organisation>> GetOrgansiationsByPayeRef(string pa
             return json == null ? null : JsonConvert.DeserializeObject<IEnumerable<Organisation>>(json);
         }
 
+        public async Task<IEnumerable<Organisation>> GetOrgansiationsByAorn(string aorn, string payeRef)
+        {
+            var baseUrl = GetBaseUrl();
+            var url = $"{baseUrl}api/pensionsregulator?payeRef={HttpUtility.UrlEncode(payeRef)}&aorn={HttpUtility.UrlEncode(aorn)}";
```
Comment (Howesy77) [inline src/SFA.DAS.EmployerAccounts/Services/PensionRegulatorService.cs]: Wrong....

**Pull request merged**